client/web/antrea-ui/src/routes/summary.tsx

```diff
@@ -75,29 +85,155 @@ function agentPropertyValues(agent: AgentInfo): string[] {
 }
 
 function ComponentSummary<T>(props: {title: string, data: T[], propertyNames: Property[], getProperties: (x: T) => string[]}) {
+    const itemsPerPage = 10; // Display 10 items per page for Agents table
+    const isAgentTable = props.title === 'Agents';
+
+    const [currentPage, setCurrentPage] = useState(1);
+    const [sortConfig, setSortConfig] = useState<SortConfig | null>(null);
```

Comment:
assuming my understanding of the code is correct, by default there is no sorting (which is the current behavior)?

Reply:
@Rupam-It please confirm